functions.py

```diff
@@ -47,7 +50,7 @@ def get_all_repos(username):
     all_repos = redis_client.get(f"{username}"+"_all_repos")
     if all_repos is None:
         try:
-            g = Github(base_url="https://api.github.com")
+            g = Github(base_url=URL)
             all_repos = []
             user = g.get_user(username)
             repos = user.get_repos()
```

Comment:
**Suggestion:** The get_all_repos function should return the list of all repositories. Currently, it does not return anything.
```suggestion
            # Return the list of repositories
            repos = user.get_repos()
            return repos
```